qiskit/algorithms/evolvers/variational/variational_principles/real_mc_lachlan_principle.py

```diff
@@ -43,27 +42,17 @@ class RealMcLachlanPrinciple(RealVariationalPrinciple):
     means that we consider real time dynamics.
     """
 
-    def calc_metric_tensor(
+    def create_qfi(
```

Comment:
Could we avoid this function and instead directly use the `QFI` method? (If not, maybe a property would be more intuitive since this doesn't really do much work? 🙂)